bliss/encoder.py

```diff
@@ -179,8 +179,6 @@ def encode_batch(self, batch):
 
         assert inputs.size(2) % 16 == 0, "image dims must be multiples of 16"
         assert inputs.size(3) % 16 == 0, "image dims must be multiples of 16"
-        bn_warn = "batchnorm training requires a larger batch. did you mean to use eval mode?"
-        assert (not self.training) or batch["images"].size(0) > 4, bn_warn
```

Comment:
Do we not need these anymore?

Reply:
No longer. I probably shouldn't have added these in the first place.